src/components/organisms/UiMultipleChoices/UiMultipleChoices.vue

```diff
@@ -146,27 +145,10 @@ const updateHandler = (newValue: MultipleChoicesModelValue, index: number) => {
 };
 
 function focusInvalidChoice() {
-  if (invalidInputsRefs.size > 0) {
-    const element = [ ...invalidInputsRefs.values() ][0];
-    focusElement(element, true);
-  }
+  const firstInvalidChoice = multipleChoicesItemRefs.value.find((element, index) => hasError(index));
+  if (firstInvalidChoice) focusElement(firstInvalidChoice.$el.querySelector('input'), true);
```

Comment:
Using `querySelector` is not desirable, but I think it still beats `.content[0].content.input`. 
1. It's unobvious what such path is targeting
2. If slots are used, this path can be incorrect, while `.querySelector('input')` is fairly sure to be there

Reply:
ok, for me we should use refs instead `querySelectors`. I agree with both of your points. But... we definitely need to develop an API in CL that will solve these problems and it will not be "querySelector"

Reply:
I got an idea to use something like this:
```ts
export const deepSearchObjectByKey = (obj: object, key: string) : { found: boolean, value?: unknown } => {
  if (key in obj) {
    return {
      found: true,
      value: obj[key as keyof typeof obj],
    };
  }

  // eslint-disable-next-line no-restricted-syntax
  for (const property of Object.values(obj).filter((value) => value && typeof value === 'object')) {
    const result = deepSearchObjectByKey(property, key);
    if (result.found) return result;
  }

  return { found: false };
};
```

To search for 'input', but it seems like this is the only place with such nested reach, so it might be a bit of a waste. 🤔 
Additionally in case of using <slots> it's unlikely that refs would be used correctly anyway.